src/main/java/com/runtimeverification/mvnplugin/ErrorCodesMojo.java

```diff
@@ -91,7 +91,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         return getCSVMap(csvFiles);
     }
 
-    private static final String PREFIX_PATTERN = "(UB|CV|SE|USP|IMPL|L|URV|DRV|CND)";
+    private static final String PREFIX_PATTERN = "([A-Z]{2,})";
```

Comment:
is this going to lead to false positives though?

Reply:
It's only applied to the contents `Error_Codes.csv` and to the filenames in the `examples` directory where it doesn't seem to result in any false positives.

Reply:
alright.